Source/WebCore/rendering/svg/SVGTextMetricsBuilder.cpp

```diff
@@ -76,9 +76,26 @@ void SVGTextMetricsBuilder::advanceSimpleText()
 void SVGTextMetricsBuilder::advanceComplexText()
 {
     unsigned metricsLength = currentCharacterStartsSurrogatePair() ? 2 : 1;
+
+    // FIXME: NON-COCOA platforms shifts some pixels for SVGText when x,y, and rotate are used.
```

Comment:
You can reference the bugzilla you have created in this FIXME. There is a useful short url format: `webkit.org/b/<bugzilla-id>`